noted/notes/v1/notes.proto

```diff
@@ -17,6 +17,12 @@ message Note {
     repeated Block blocks = 4;
 }
 
+message MetaNote {
+    string id = 1;
+    string author_id = 2;
+    string title = 3;
+}
+
```

Comment:
I think `NoteMetadata` is more appropriate. Also maybe you don't really need to create this second data type. You can just return an array of `Note`s and not include the `blocks` (leave it to `nil`). LMKWYT.

Reply:
Yes you're right i will do this ! (I already did it like that) Thanks